kombu/transport/azureservicebus.py

```diff
@@ -229,7 +227,7 @@ def _delete(self, queue: str, *args, **kwargs) -> None:
         """Delete queue by name."""
         queue = self.entity_name(self.queue_name_prefix + queue)
 
-        self._queue_mgmt_service.delete_queue(queue)
+        self.queue_mgmt_service.delete_queue(queue)
```

Comment:
This is the change that I called out in the PR description. I'm not sure if this was accessing the variable instead of using the method intentionally or was an accident.